src/engraving/dom/articulation.cpp

```diff
@@ -608,6 +610,27 @@ String Articulation::accessibleInfo() const
     return String(u"%1: %2").arg(EngravingItem::accessibleInfo(), translatedTypeUserName());
 }
 
+bool Articulation::leaveSpaveForTie() const
```

Comment:
I wonder a little bit if this code doesn't belong in chordlayout.cpp, especially since that is the only place where it is called. Is there a specific reason to put it here?

Reply:
Also good point 👍